### PR TITLE
feat(web): integrate rocket.chat

### PR DESCRIPTION
**Pull request by chandhuDev**
It integrates the Rocket.Chat webhook to current notification system.
And fixes the bug in the deployment by fixing the NotificationMessageTypes to align exactly with the file names in the `web/crux/assets/notificationTemplates`

### CODE REVIEW

File: web/crux/src/domain/notification-templates.ts [hunk 1]
```diff
@@ -4,7 +4,7 @@ import { CruxInternalServerErrorException } from 'src/exception/crux-exception'
 
 const title = 'dyrector.io'
 
-export type NotificationMessageType = 'node' | 'version' | 'invite' | 'failed-deploy' | 'successful-deploy'
+export type NotificationMessageType = 'node' | 'version' | 'invite' | 'failedDeploy' | 'successfulDeploy'
```
Comment (robot9706) [inline web/crux/src/domain/notification-templates.ts:7]: I think kebab-case is better here. Could you rename the template files instead?

**Review by robot9706 — CHANGES_REQUESTED** [review]
Great work, I've tested it with a local rocket chat instance and it works fine.

There's one other thing though: You modified the database schema so please add a migration script as well, you can do it using `npx prisma migrate dev`.

**Comment by chandhuDev** [comment]
can you please check now @robot9706

**Comment by polaroi8d** [comment]
> can you please check now @robot9706

Linter is still failing.

**Pull request merged**